### ISSUE TITLE
Added Life Sense

### ISSUE THREAD

**Issue by Galphanore** [dev7355608/vision-5e#84]
## Requested Feature

The ability to add custom vision modes. The existing vision modes in the module are awesome but they're all just slightly off from what I want. It would be nice to have a section in the module settings to be able to just select tags and features of the vision mode. A dropdown for PC/NPC actors vs Tokens, notes, and door controls. What conditions (burrowing, defeated, etc) disable the vision, and whether it's blocked by walls. 

### Alternative Option

The specific use case I'm looking for is a vision mode that detects PC/NPC actors that are not objects through walls, ignores darkness, and cannot detect tokens that are dead. It would be configured automatically for PC actors with the **Soul Sight** feat. 

Obviously this is pretty obscure and almost entirely exclusive to my table so I thought the customizable vision modes would be a better option. Especially since several other existing feature requests are asking for equally obscure homebrew vision types.

**Comment by dev7355608** [comment]
Creating UI to allow customizing detection modes behavior or the creation of new ones isn't a simple thing that I don't have the time for. Also creating/modifying a new detection modes is fairly simple and quick with the API (assuming some basic scripting knowledge).

I added *Life Sense* in Vision 5e 2.2.0 (https://github.com/dev7355608/vision-5e/releases/tag/v2.2.0), which is probably close enough to your *Soul Sight*. It's a monster feat in official content that at least one monster has.
>  A creature with *Life Sense* can sense the location of any creature within range, unless that creature's type is construct or undead.

**Comment by dev7355608** [comment]
Unfortunately it isn't as simple as copy & paste to make this work in V11. It may also depend on the dnd5e system version you're on. I'd need to spent some time to backport it which I currently don't have.

**Comment by Galphanore** [comment]
Ok, thanks.
